main.go

```diff
@@ -27,8 +39,14 @@ func main() {
 		StatFn:  runhttp.StatFromContext,
 		Storage: &nopStorage{},
 	}
+	fetch := &v1.CloudFetchHandler{
+		LogFn:   runhttp.LoggerFromContext,
```

Comment:
For the sake of consistency, I'd recommend having an alias in `domain` that defines these context functions like we do for the `LogFn` and `Log` types. This way there aren't direct references to `runhttp` outside of `domain`.

Reply:
I'll replace the usage of `runhttp` with `domain` aliases.